src/components/Button.js

```diff
@@ -14,15 +14,17 @@ class Button extends React.Component {
   onButtonLeave = () => this.setState({ style: STYLE.BUTTON.standard });
 
   render() {
-    const { customType } = this.props;
-    const { style } = this.state;
+    const { customType, onClickAction, isDisabled } = this.props;
+    let { style } = this.state;
+    if (isDisabled) style = STYLE.BUTTON.disabled;
```

Comment:
Add curly braces.
`if (isDisabled) {
  style = STYLE.BUTTON.disabled
}`

and only change values in state by using the setState method (avoid mutating any state property)